content/en/software/install-and-run/running-on-win.md

```diff
@@ -7,43 +7,15 @@ aliases:
+Medley "native" works within the ordinary Windows environment. Installation uses standard Windows installer and you start Medley from a standard Command (or Powershell) window.  Medley 'native' is best for users who prefer to stay (almost) exclusively in the Windows environment.  
  
-Medley on WSL and 'native' Medley both require 64-bit Windows 10/11.
-
-
-### **1.  Running Medley Online**
-
-Instructions on running Medley Online can be found [here](/software/access-online/)
-
-### **2.  Running on Windows with WSL**
-
-Medley will run on either WSL1 or WSL2.  WSL2 is preferred, but for older machines that do not support virtualization (see [here](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/reference/hyper-v-requirements)) or Windows builds prior to Windows 10 Build 19041, WSL1 will work just fine although it will be limited to the VNC mode (see below).
-
-When running under WSL2, Medley can display in one of two ways: in an X-Window (using the X Windows server built-in to WSL2) or in a VNC-viewer Window.  Although the X-windows approach is simpler (i.e., behind the scenes), it does not scale well on high-DPI displays.  In VNC mode, the Medley window will scale according to the Windows Display Scale settings.  VNC mode is set using the `--vnc` flag on the `medley` command line.  In the absence of the --vnc flag,  X Windows mode will be used.  When running on WSL1, VNC mode is always used.
-
-#### To install and run Medley with WSL
-
-1. *Install WSL:* See [here](https://learn.microsoft.com/en-us/windows/wsl/install) for instructions on installing WSL.
-
-2. *Install Medley within WSL:* Once WSL is installed, open a terminal to WSL (e.g., by typing "wsl" in a Command or Powershell window) and follow the instructions for installing and running Medley on WSL and standard Linux that can be found [here](/software/install-and-run/running-on-linux). 
-
-3. *Run Medley:* Once Medley has been installed on WSL, you can run Medley from either a WSL terminal as described in the Running Medley on Linux instructions ([here](/software/install-and-run/running-on-linux)) or by typing `wsl medley <flags and options>` in a Command or Powershell window.
-
-    Documentation for the `<flags and options>` to the `medley` command can be found [here](https://online.interlisp.org/downloads/man_medley.html)
-
-    For first-time users: `wsl medley --vnc --apps --interlisp --noscroll` or, equivalently, `wsl medley -v -a -e -n` is a good starting point.  This will give you a fully populated Medley system, including the applications built on Medley such as Notecards and Rooms.
-
+Medley on WSL and "native"' Medley both require 64-bit Windows 10/11.
 
-### **3.  Running on Windows 'natively'**
+### Running on Windows 'natively'**
```

Comment:
Change to "natively", double quotes for consistency.  

Need one more asterisk to end the heading properly.

Reply:
This is resolved, "natively" is now consistently unquoted.